demo/src/style/app.scss

```diff
@@ -89,6 +89,11 @@ p.github-buttons > iframe {
   cursor: pointer;
 }
 
+.bd-sidebar .nav > li > a.active{
```

Comment:
that's not his code - that was mine originally.  also i neglected to put newlines on some compound css defs.